umm-spec-lib/resources/json-schemas/variable/umm/v1.9/umm-var-json-schema.json

```diff
@@ -0,0 +1,604 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "UMM-Var",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "Name": {
+      "description": "The name of a variable.",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 256
+    },
+    "StandardName": {
+      "description": "This is the more formal or scientific name, .e.g., the CF Standard Name.",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 256
+    },
+    "AdditionalIdentifiers": {
+      "description": "Any additional identifiers of a variable.",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/AdditionalIdentifierType"
+      },
+      "minItems": 1
+    },
+    "LongName": {
+      "description": "The expanded or long name related to the variable Name.",
+      "$ref": "#/definitions/LongNameType"
+    },
+    "Definition": {
+      "description": "The definition of the variable.",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 10000
+    },
+    "Units": {
+      "description": "The units associated with a variable.",
+      "type": "string",
+      "minLength": 0,
```

Comment:
does this mean you can have an empty unit? Does 1 mean "you have to give something if you define the field, but you can still NOT define the field"?

Reply:
Yes, it has been in existence 6 years from UMM-Var's inception.  There are a lot of other cases, that I have fixed in the past, and there are still other issues.  Its because not everyone who makes changes to JSON schemas understands what each thing means. A lot of people mistake the minLength or minItems to mean required.  I have changed the 0 to a 1.